scipy/signal/signaltools.py

```diff
@@ -592,7 +596,7 @@ def choose_conv_method(in1, in2, mode='full', measure=False):
 
     Notes
     -----
-    For large n, ``measure=False`` is accurate and can quickly determine the
+    For large n, `measure=False` is accurate and can quickly determine the
```

Comment:
This should be

```
For large n, ``measure=False`` is accurate 
```

In RST, it's double backticks for code, single backticks for variables or functions